ui/imports/shared/popups/walletconnect/ConnectDAppModal.qml

```diff
@@ -93,56 +91,64 @@ StatusDialog {
     signal disconnect()
 
     width: 480
-    implicitHeight: !d.connectionAttempted ? 633 : 681
```

Comment:
Based on [ConnectING UI](https://www.figma.com/design/HrmZp1y4S77QJezRFRl6ku/dApp-Interactions---Milestone-1?node-id=481-161782&t=nJ1UAEBwyN4zBqKS-0) and [ConnectED UI](https://www.figma.com/design/HrmZp1y4S77QJezRFRl6ku/dApp-Interactions---Milestone-1?node-id=481-165751&t=lw4lban8exsantDZ-0)
maybe it would make sense to leave `implicitHeight` with those values as is?

Reply:
No, no hardcoded popup heights; this allows the popup to freely shrink and grow (within the `margins` limits)